internal/wire/wire_test.go

```diff
@@ -138,10 +139,11 @@ func unloadTestData(context interface{}, db *db.DB) error {
 		tx.RemoveQuad(quad)
 	}
 
-	store, err := cayleyshelf.New(cfg.MustURL("MONGO_URI").String())
+	store, err := cayleyshelf.New(cfg.MustURL("MONGO_URI").String(), nil)
```

Comment:
You'll need to use the same "session name" as the mongo session that was registered, in this case `tests.TestSession`.